solutions/src/test/java/ru/sberbank/school/task09/RouteServiceImplTest.java

```diff
@@ -0,0 +1,94 @@
+package ru.sberbank.school.task09;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import ru.sberbank.school.task09.kryo.KryoSerializeManager;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.File;
+import java.util.ArrayList;
+
+class RouteServiceImplTest {
+
+    private static RouteService service;
+    private static String directory;
+    private static Route<City> saved;
+    private static Route<City> loaded;
+
+    @BeforeAll
+    @SuppressWarnings("unchecked")
+    static void init() {
+        directory = "F:\\Task09Cache";
+        service = new RouteServiceImpl(directory, new KryoSerializeManager(directory));
+        saved = service.getRoute("Moscow", "Novosibirsk");
+        loaded = service.getRoute("Moscow", "Novosibirsk");
+    }
+
+    @AfterAll
+    static void cleanUp(){
```

Comment:
WhitespaceAround: &#39;{&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-244935536>*